build/storage/python_system_tools/config.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+from pathlib import Path
+from consts import SHARE_DIR_PATH
+
+path = Path.cwd()
+data_path = path.parent / "python_system_tools/config.json"
+with open(file=data_path) as f:
+    data = json.load(f)
+
+ipu_storage_address = data["ipu_storage_container_address"]
+ipu_storage_image = data["ipu_storage_container_docker_image"]
+
+storage_address = data["storage_target_address"]
+storage_image = data["storage_target_docker_image"]
+
+cmd_sender_address = data["cmd_sender_address"]
+cmd_sender_image = data["cmd_sender_docker_image"]
+
+user_name = data["user"]
+password = data['password']
```

Comment:
To be honest I am not sure that specifying passwords in a file it is a right way to do this. Will it be better to stick to the idea of passwordless ssh? At least for now?